src/borg/archiver.py

```diff
@@ -2857,7 +2857,7 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
                                help='print statistics for the created archive')
         subparser.add_argument('--list', dest='output_list', action='store_true',
                                help='output verbose list of items (files, dirs, ...)')
-        subparser.add_argument('--filter', metavar='STATUSCHARS', dest='output_filter',
+        subparser.add_argument('--filter', metavar='AMUEdbchsfi-x', dest='output_filter',
```

Comment:
hmm, is this really an improvement or just another place that would need to get updated and could easily be forgotten?

the stuff is already described in 2819+.